content/en/templates/taxonomy-templates.md

```diff
@@ -126,9 +126,13 @@ Taxonomies can be ordered by either alphabetical key or by the number of content
 
 ```
 <ul>
-  {{ $data := .Data }}
+  {{ $type := .Type }}
   {{ range $key, $value := .Data.Terms.Alphabetical }}
-  <li><a href="{{ $.Site.LanguagePrefix }}/{{ $data.Plural }}/{{ $value.Name | urlize }}"> {{ $value.Name }} </a> {{ $value.Count }} </li>
+  {{ $name := .Name }}
```

Comment:
Indent everything under range by 4 spaces.

Reply:
Fixed all indentation, but I am using 2 spaces since the original indentation seems to be 2-space. Please comment again if anything needs to be fixed. Thanks!